### ISSUE TITLE
Error loading pokemon list bad authentication

### ISSUE THREAD

**Issue by drsweg** [Ar1i/PokemonGo-Bot#2416]
Any ideas as to why? It all started last night and still cannot get it to work

**Comment by xRealMaks** [comment]
Maybe your username/password is wrong..
*Just my first idea as I read bad authentication...

**Comment by drsweg** [comment]
Thats what i thought but i entered them right i even changed my password and nothing. Uninstalled and reinstalled countless times

**Comment by xRealMaks** [comment]
Can you try to login in the offical PokemonGO app?

**Comment by drsweg** [comment]
Im still logged on my phone it seems to work so im stumped..

**Comment by xRealMaks** [comment]
Can you relog in the app?

**Comment by drsweg** [comment]
Yeah i logged in no problem. I tried to log my friend in yesterday than all of a sudden it did the error loading pokemon list

**Comment by xRealMaks** [comment]
Can you try to start the bot with another account?

**Comment by drsweg** [comment]
Thats what started it i think. My friend tried logging in and the error
occurred than i tried mine and the same thing happened

**Comment by Hydro420** [comment]
Are you using BlueStacks by any chance?

You are able to have more than one goggle account at a time on it & if you do, PokemonGO will not start. I had the same issue.
Go to settings and make sure there is only one goggle account and that its the account you are playing PokemonGO with.

**Comment by xRealMaks** [comment]
Maybe your firewall blocks the connection from the bot..

**Comment by drsweg** [comment]
I use bluestacks some times but not at the same time as the bot.

**Comment by drsweg** [comment]
I put the program on the allowed list

**Comment by Hydro420** [comment]
Well you should never do that in the first place... What I meant is, go under bluestacks settings and make sure you only have one google account linked. If you have more than one it wont let you login to PokemonGO.

**Comment by drsweg** [comment]
I will do that as soon as i get home hopefully this works

**Comment by Hydro420** [comment]
Hit the "Google." & make sure there is only one account. Use the account you are playing pokemonGO with or again, it wont let you login.

![untitled3](https://cloud.githubusercontent.com/assets/20824888/17535093/2a45903a-5e52-11e6-9a60-6285ac07690a.png)

**Comment by drsweg** [comment]
So pokecrot should let me login as long as there's only one account on my bluestacks?

**Comment by Hydro420** [comment]
Did you just say pokecrot....... If I know of anything, thats the guy who stole Ar1i's bot.. which is on here...?

**Comment by drsweg** [comment]
Well Ar1i's bot i meant

**Comment by drsweg** [comment]
Let me rephrase the bot should work properly as long as there's only one account on my bluestacks?

**Comment by eLDeMe** [comment]
can you show a screen capture of the problem?

**Comment by drsweg** [comment]
![20160809_180932](https://cloud.githubusercontent.com/assets/20933416/17535450/8b43aa84-5e5c-11e6-9891-8f6e92afb326.jpg)

**Comment by xRealMaks** [comment]
can you do another screenshot without motion-blur? :smile: 
Maybe use snipping tool and upload it to imgur.com

**Comment by eLDeMe** [comment]
those errors apear when servers are unstable or down, i get them some times, but not a problem, try to push the force unban button just in case and wait a few minutes

**Comment by eLDeMe** [comment]
if I shut down my router i'll get those message because no connection, i don't know if I can explain myself

**Comment by drsweg** [comment]
![screenshot 1](https://cloud.githubusercontent.com/assets/20933416/17535737/66d940c6-5e5e-11e6-9b83-daccf6a3e1a4.png)

**Comment by eLDeMe** [comment]
and an old version, maybe before the api change issue

**Comment by eLDeMe** [comment]
@xRealMaks should close this

**Comment by drsweg** [comment]
That was my problem i was using pokecrot im using ar1i's bot  now and bluestacks was still running in the background it works now!

**Comment by xRealMaks** [comment]
Delete pokecrot please. The "dev" copied the bot. (the old versions)
